docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -1999,6 +1999,45 @@ use custom implementations of the following components:
 When creating custom implementations of Elytron components, they must
 present the appropriate capabilities and requirements.
 
+=== Default SSLContext
+
+Many libraries that can be used within deployments may require SSL configuration for any connections they establish, these libraries tend to be configurable by the caller or if no configuration is provided fall back to using the default `SSLContext` for the process available from: -
+
+[source, java]
+----
+javax.net.ssl.SSLContext.getDefault();
+----
+
+By default this `SSLContext` is configured using system properties, however within the WildFly Elytron subsystem it is possible to specify that one of the configured contexts should be associated and used as the default.
+
+To make use of this feature configure your `SSLContext` as normal, the following command can then be used to specify which `SSLContext` should be used as the default.
+
+[source, ruby]
+----
+/subsystem=elytron:write-attribute(name=default-ssl-context, value=client-context)
```

Comment:
Looking into implementation. Does it mean restart-required will be outputed here? I understand model does not provide this level of configuration. But maybe we should document this mismatch and despite restart-required is printed reload is enough in this case.

Reply:
I see correctly reload required after writing to attribute
```    "outcome" => "success",
    "response-headers" => {
        "operation-requires-reload" => true,
        "process-state" => "reload-required"
    }
```

Reply:
+1 I didn't add that as we don't tend to all out every reload required situation in the documentation but the restart required was a special case so needed a special mention.